py-base/environment.yml

```diff
@@ -94,3 +102,8 @@ dependencies:
   - xgcm
   - xlrd
   - zarr
+  - pip:
+    # PyPI package is older, 
+    # but installing from github leads to conda-lock error
+    # - git+https://github.com/open-ocean-sounding/echopy.git
+    - echopy
```

Comment:
This package does not install its dependencies and pip installing it will bring a broken pkg, or not fully functioning at best. I sent https://github.com/conda-forge/staged-recipes/pull/22020 PR to help out a bit with its installation. It is only missing one optional dependency as far as I can tell.

Reply:
Thank you!!!